packages/web-app-files/src/helpers/share/role.ts

```diff
@@ -278,6 +282,24 @@ export abstract class PeopleShareRoles {
       .find((r) => r.folder === isFolder && r.bitmask(allowSharing) === bitmask)
     return role || this.custom(isFolder)
   }
+
+  static listByBitmask(
```

Comment:
Shouldn't a bitmask result in exactly one role? Afair for that reason I built only `getByBitmask` and not `listByBitmask`.

Reply:
Maybe bitmask is the wrong wording then 🤔 

Let's say I have a share that has the permissions 25 (read, share, delete). Then the method `getByBitmask` only returns the one role having exactly these permissions. Whereas here, I want to have all roles that have permissions wich are "inside". Meaning I want the viewer role (and a potential "Viewer+Share(+Delete)" role, if it would exist).